app/views/trains/index.html.slim

```diff
@@ -1,5 +1,7 @@
 h1 Trains Index
 = link_to 'New train', new_train_path
+br
+= notice
```

Comment:
can you use the same styling as the error alerts so that its got a cleaner ui
https://getbootstrap.com/docs/5.0/components/alerts/

just make sure to use the success one for this, since its a positive message not a red negative one